client/src/pages/Profile/Profile.tsx

```diff
@@ -0,0 +1,140 @@
+import { useAppData } from "store/Store"
+import { useNavigate } from "react-router-dom"
+
+import ProfileInformationPanel from "components/generic/ProfileInformationPanel/ProfileInformationPanel"
+import FullPageBackgroundImage from "components/generic/FullPageBackgroundImage/FullPageBackgroundImage"
+import { Footer } from "components/generic/Footer/Footer"
+
+const SignOutButton = () => {
+  const navigate = useNavigate()
+  const handleOnclick = () => {
+    navigate("/login")
+  }
+  return (
+    <div
+      className="border-red space-x-4; disabled:bg-gray-3 text-red hover:bg-red
+    flex flex-col items-center rounded-md border bg-white px-8 py-2 font-sans font-bold 
+   hover:text-white enabled:border"
+    >
+      <button className="uppercase" onClick={handleOnclick}>
+        sign out
+      </button>
+    </div>
+  )
+}
+
+const Field = ({
+  subtitle,
+  description
+}: {
+  subtitle: string
+  description?: string
+}) => {
+  return (
+    <>
+      <div>
+        <p className="pb-1 text-base font-normal leading-tight text-stone-300">
+          {subtitle}
+        </p>
+        <p className="text-black/opacity-20 pb-1 text-base font-normal leading-tight">
+          {description}
+        </p>
+      </div>
+    </>
+  )
+}
+export default function Profile() {
+  const [{ currentUserData }] = useAppData()
+  const [{ currentUser }] = useAppData()
+
+  function toDateTime(secs?: number) {
+    const t = new Date(1970, 0, 1) // Epoch
```

Comment:
is this right?????

Reply:
I thought the default date is 1970, 01, 01. And you add seconds to it. Will change it